spec/requests/attachments_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.describe "Attachments" do
```

Comment:
This comment is rather subjective, so I'll just write it and leave any changes up to you. It takes a lot of time to run the tests already - especially request and feature tests - and I'm worried about incrementally making that worse. It's not something we've really documented; personally, I only create new test files when I'm missing coverage for some reason - it just feels like extra effort to me.

Reply:
I'll change this to a view test.